### PR TITLE
Accelerated container image for AMD

### PR DESCRIPTION
**Pull request by kwozyman**
I've added the first bootc bare metal accelerated bootc container image and it's README.

### CODE REVIEW

File: training/Makefile [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+REGISTRY_ORG ?= ai-lab
+IMAGE_NAME ?= amd
+IMAGE_TAG ?= latest
+
+CONTAINER_TOOL ?= podman
+CONTAINER_TOOL_EXTRA_ARGS ?=
+
+.PHONY: amd
+amd:
+	make -C amd/ image
```
Comment (rhatdan) [inline training/Makefile:13]: I think you would need to change this to
make -C amd/ image FROM=$(FROM) REGISTRY=$(REGISTRY) REGISTRY_ORG=$(REGISTRY_ORG)...

File: training/amd/Makefile [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+CONTAINER_TOOL ?= podman
+CONTAINER_TOOL_EXTRA_ARGS ?=
+
+.PHONY: image
+image:
+	"${CONTAINER_TOOL}" build \
+		--file Containerfile \
+		--tag "${REGISTRY}/${REGISTRY_ORG}/${IMAGE_NAME}:${IMAGE_TAG}" \
+		--build-arg BASEIMAGE=${FROM} \
```
Comment (rhatdan) [inline training/amd/Makefile]: In the other repos we have been using 

--from $(FROM) rather then --build-arg.

File: training/amd/Containerfile [hunk 3]
```diff
@@ -0,0 +1,8 @@
+ARG BASEIMAGE="quay.io/centos-bootc/centos-bootc:stream9"
+
+FROM ${BASEIMAGE}
```
Comment (rhatdan) [inline training/amd/Containerfile]: I would prefer to use `FROM quay.io/centos-bootc/centos-bootc:stream9` and use `--from $(FROM) in the Makefile like we do in the rest of the repo.

**Comment by rhatdan** [comment]
Looks good, just a couple of nits.

File: training/Makefile (hunk 1 above)
Comment (kwozyman) [reply training/Makefile:13]: I think these get inherited as environment variables. At least on my Fedora 39 a `make amd REGISTRY=foo` gets me the correct name just with the above

File: training/amd/Containerfile (hunk 3 above)
Comment (kwozyman) [reply training/amd/Containerfile]: Done

File: training/amd/Makefile (hunk 2 above)
Comment (kwozyman) [reply training/amd/Makefile]: Done

File: training/Makefile (hunk 1 above)
Comment (rhatdan) [reply training/Makefile:13]: If they do that is great, ignore my comment.

**Comment by rhatdan** [comment]
@kwozyman you need to sign your commits, and then we can merge.

**Comment by rhatdan** [comment]
LGTM

**Pull request merged**